python/doc/examples/calibration/least_squares_and_gaussian_calibration/plot_calibration_logistic.py

```diff
@@ -116,22 +224,18 @@ def logisticModel(X):
 # -------------------------------------
 
 # %%
-timeObservationsVector = ot.Sample([[timeObservations[i, 0] for i in range(nbobs)]])
-timeObservationsVector[0:10]
-
-# %%
-populationObservationsVector = ot.Sample(
-    [[populationObservations[i, 0] for i in range(nbobs)]]
-)
-populationObservationsVector[0:10]
-
-# %%
-# The reference values of the parameters.
+# The calibration algorithm takes input and output samples as arguments.
+# In this case, we choose to consider one single observation in dimension 22.
+# In order to perform calibration, we create a `Sample` of input times which has
+# one observation in dimension 22.
+timeObservationsSample = ot.Sample([timeObservations.asPoint()])
+timeObservationsSample[0, 0:5]
 
 # %%
-a = 0.03134
-c = -22.58
-thetaPrior = [a, c]
+# Similarly, we create a `Sample` of output populations which has
+# one observation in dimension 22.
+populationObservationsSample = ot.Sample([populationObservations.asPoint()])
+populationObservationsSample[0, 0:5]
```

Comment:
```suggestion
print(populationObservationsSample[0, 0:5])
```
This might be the reason for the weird blank spaces: Sphinx confuses the output with RST text if it is not clearly labeled as a print.

This change should be made here and at multiple other places.

Reply:
Ok. Fixed.

Reply:
It took some time, but I soon realized that these blank spaces were almost everywhere in the code. Fixed.